versions.yml

```diff
@@ -236,7 +288,63 @@ modGroups:
       url: https://cdn.modrinth.com/data/PtjYWJkn/versions/zgaajBnP/sodium-extra-0.5.2%2Bmc1.20.2-build.113.jar
       requires: [Sodium]
 
-  # Developer Tools:
+    - name: Blur
+      license: MIT
+      homepage: https://modrinth.com/mod/blur-fabric
+      modrinthId: NK39zBp2
+      desc: Adds immersive blur to GUI backgrounds
+      url: https://cdn.modrinth.com/data/NK39zBp2/versions/qh7RxICc/blur-3.1.1.jar
+
+    - name: Borderless Mining
+      license: MIT
+      homepage: https://modrinth.com/mod/borderless-mining
+      modrinthId: kYq5qkSL
+      desc: Changes the fullscreen option to use a borderless window that fills the screen.
+      url: https://cdn.modrinth.com/data/kYq5qkSL/versions/lhrW53q7/borderless-mining-1.1.8%2B1.20.1.jar
+
+    - name: Zoomify
+      license: LGPL-3
+      homepage: https://modrinth.com/mod/zoomify
+      modrinthId: w7ThoJFB
+      desc: Adds a zoom key with many customization options.
+      url: https://cdn.modrinth.com/data/w7ThoJFB/versions/QRQLkNPJ/Zoomify-2.12.0.jar
+      incompatibleWith: [Logical Zoom]
+      requires: [FabricKotlinLanguage, Yet Another Config Lib]
+
+    - name: ClickThrough
+      license: MIT
+      homepage: https://modrinth.com/mod/clickthrough
+      modrinthId: Z5b0cAlD
+      desc: Click through signs and item frames to chests
+      url: https://cdn.modrinth.com/data/Z5b0cAlD/versions/bwQT3Pqa/clickthrough-1.20-fabric0.83.0-0.4.1.jar
```

Comment:
Does it work on 1.20.2?

Reply:
This is the only mod that I didn't test on Wynncraft as I said in the main comment as I couldn't find any spot where it could be tested, but I did test it on singleplayer where there are no issues there, so I decided to update it

Reply:
Forgot to ask. If that is good enough for you, or should I remove that mod?

Reply:
If it doesnt crash it is good enough for me.